### ISSUE TITLE
No handling magnet links

### ISSUE THREAD

**Issue by worphworph** [truefedex/tv-bro#97]
Great browser but, I'm getting "net::ERR_UNKNOWN_URL_SCHEME" when trying click on magnet link button (thepiratebay.org). Newest version TV bro, biglyBT for android, and nVidia shield user. Many thanks for response.

**Comment by MailYouLater** [comment]
Possibly related: #14

**Comment by ggg70** [comment]
I suppose you updated to Android 11 and there are some changes for WebView. I can confirm the same behavior on Realme TV Stick 4K with intent:// links. Some solutions are presented here [https://github.com/auth0/auth0-cordova/issues/180](url) or here [https://www.androidbugfix.com/2021/11/android-webview-neterrunknownurlscheme.html](url)

**Comment by truefedex** [comment]
Probably fixed in release (please reopen if not):
https://github.com/truefedex/tv-bro/releases/tag/v1.8.5

**Comment by truefedex** [comment]
There are reports that magnet links are not opening again. But I can't reproduce this. If anyone can provide an example of a page with such a link, please add it here.

**Comment by ggg70** [comment]
No, sorry, my issue is with _intent://_ links. I thought it was caused by the same problem that previously cased magnet: links to not work, but now seems a different cause (that's why I corrected my previous post and linked to the pull request)
I've just tested magnet links and indeed they open in external application (Amnis) in Android TV 11.
